pinot-common/src/main/proto/worker.proto

```diff
@@ -72,21 +72,17 @@ message StagePlan {
   int32 stageId = 1;
   string instanceId = 2;
   StageNode stageRoot = 3;
-  map<int32, StageMetadata> stageMetadata = 4;
+  map<int32, WorkerMetadata> workerMetadata = 4;
 }
 
-message StageMetadata {
-  repeated string instances = 1;
-  repeated string dataSources = 2;
-  map<string, SegmentMap> instanceToSegmentMap = 3;
+message WorkerMetadata {
+  string instances = 1;
+  string dataSources = 2;
+  map<string, SegmentList> instanceToSegment = 3;
```

Comment:
this is no longer needed to point from instance to table type to segments, b/c worker metadata contextually only work on 1 worker